conda/base/context.py

```diff
@@ -30,6 +31,16 @@
 except ImportError:  # pragma: no cover
     from .._vendor.toolz.itertoolz import concat, concatv, unique
 
+try:
+    os.getcwd()
+except (IOError, OSError) as e:
+    if e.errno == ENOENT:
+        # FileNotFoundError can occur when cwd has been deleted out from underneath the process.
+        # To resolve #6584, let's go with setting cwd to $HOME, and see how far we get.
+        os.chdir(expand('~'))
```

Comment:
What if the user has no home directory? (Service account or deployment user or something similar)

Should we change it to a directory which is guaranteed to exist? `sys.prefix` maybe?

Reply:
I think that's a great idea.